HpToolsLauncher/Runners/AlmTestSetsRunner.cs

```diff
@@ -938,17 +941,16 @@ public bool ValidateListOfParameters(string paramsString, string[] parameters, L
         /// Validates test parameters
         /// </summary>
         /// <param name="param"></param>
-        /// <param name="parameterList"></param>
-        /// <param name="isParameter"></param>
+        /// <param name="paramList"></param>
         /// <returns>true if parameter is valid, false otherwise</returns>
-        public bool ValidateParameters(string param, List<string> parameterList, bool isParameter)
+        public bool ValidateParams(string param, ref IList<string> paramList)
```

Comment:
The `ref` keyword here is not necessary since you don't **reassign** the variable `paramList` in this method. It is better to remove the `ref` keyword in case the variable points to another list instance by mistake.